server_installation/topics/operator/installation.adoc

```diff
@@ -0,0 +1,89 @@
+
+=== Installing {project_operator} into a cluster 
+
+To install the {project_operator} into 
+ifeval::[{project_community}==true]
+a Kubernetes cluster or
+endif::[]
+an Openshift cluster, you have two options:
+
+* The Operator Lifecycle Manager (OLM)
+* Command line installation
+
+==== Using Operator Lifecycle Manager
+
+For a new OpenShift 4.x cluster, the {project_operator} should already be available in
+OperatorHub tab of the OpenShift administration panel.
+
+image:{project_images}/operator-openshift-operatorhub.png[]
```

Comment:
Just to let you know - all this images were taken for Keycloak (community). We don't have anything specific for RHSSO. It's a chicken/egg problem so we'll probably need a new task (JIRA?) to take some new screenshots once RHSSO Operator is released.

Reply:
@slaskawi @hmlnarik Unless there is some slip, I believe the date for the SSO version is May 26 or May 29.  Is that correct?

Reply:
May 25th. Please contact with @bdaw about this offline.